<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>340</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog, QWidget {
    background-color: #f2f2f2;
    color: #202020;
    font-family: &quot;Stencil&quot;;
    font-size: 15px;
    letter-spacing: 1px;
}

/* === Заголовки === */
QLabel#labelTitle {
    font-size: 40px;
    font-weight: bold;
    color: #222222;
    text-align: center;
}

/* === Звичайні написи === */
QLabel {
    color: #303030;
    font-weight: 500;
}

/* === Поля вводу === */
QLineEdit {
    background-color: #ffffff;
    border: 2px solid #bbbbbb;
    border-radius: 6px;
    padding: 6px 10px;
    font-family: &quot;Segoue UI&quot;;
    color: #202020;
    selection-background-color: #00cc88;
    selection-color: white;
}

QLineEdit:focus {
    border: 2px solid #00cc88;
    background-color: #fcfcfc;
}

/* === Кнопки === */
QPushButton {
    background-color: #00cc88;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px 18px;
    font-family: &quot;Stencil&quot;;
    font-weight: bold;
    text-transform: uppercase;
    letter-spacing: 1px;
}

QPushButton:hover {
    background-color: #00b377;
}

QPushButton:pressed {
    background-color: #009966;
}

QPushButton:disabled {
    background-color: #cccccc;
    color: #777777;
}

/* === Чекбокс (Показати пароль) === */
QCheckBox {
    color: #303030;
    spacing: 6px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #888888;
    border-radius: 4px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #00cc88;
    border: 2px solid #00cc88;
}

QCheckBox:hover {
    color: #00aa66;
}

/* === Скролбар (на випадок, якщо є поле з прокруткою) === */
QScrollBar:vertical {
    border: none;
    background: #e0e0e0;
    width: 8px;
    margin: 0;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background: #00cc88;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #00aa66;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/resources/icons/to-do-list.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1e1e1e;              /* темний фон */
    color: #00ff88;                         /* неонова зелень */
    border: 2px solid #00ff88;
    border-radius: 8px;
    padding: 10px 18px;
    font-family: &quot;Stencil&quot;;
    font-size: 16px;
    letter-spacing: 1px;
    font-weight: bold;
    text-transform: uppercase;
    transition: all 0.2s;
}

QPushButton:hover {
    background-color: #00ff88;
    color: #101010;
}

QPushButton:pressed {
    background-color: #00cc66;
    border-color: #00cc66;
    color: #0f0f0f;
}

QPushButton:disabled {
    background-color: #333333;
    border-color: #555555;
    color: #777777;
}
</string>
       </property>
       <property name="text">
        <string>Login to System</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelUsername">
         <property name="font">
          <font>
           <family>Stencil</family>
           <pointsize>-1</pointsize>
           <fontweight>Medium</fontweight>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsername">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="font">
          <font>
           <family>Stencil</family>
           <pointsize>-1</pointsize>
           <fontweight>Medium</fontweight>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkShowPassword">
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string>STATUS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegister">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
